airbyte/mcp/cloud_ops.py

```diff
@@ -37,6 +33,7 @@ def _get_cloud_workspace() -> CloudWorkspace:
 
 
 # @app.tool()  # << deferred
+@cloud_tool(destructive=False)
```

Comment:
Let's invert the pattern in the decorator signature. Instead of 'destructive=False', let's only specify when `destructive=True`. We'll diverge our defaults from FastMCP's for the purpose of ergonomics, but the final result should be the same.